bin/cml.js

```diff
@@ -29,59 +61,52 @@ const configureLogger = (level) => {
   });
 };
 
+const setupTelemetry = async (opts) => {
+  const { cml, cmlCommand: action } = opts;
+  opts.telemetryEvent = await jitsuEventPayload({ action, cml });
+};
+
 const runPlugin = async ({ $0: executable, command }) => {
-  try {
-    if (command === undefined) throw new Error('no command');
-    const path = which.sync(`${basename(executable)}-${command}`);
-    const parameters = process.argv.slice(process.argv.indexOf(command) + 1); // HACK
-    process.exit(await pseudoexec(path, parameters));
-  } catch (error) {
-    yargs.showHelp();
```

Comment:
Do we know in what scenario this code was being hit, and would we lose without this?
Does the `.fail(handleError)` catch these?

Reply:
Yes the handleError will catch it